lib/__tests__/kv.test.js

```diff
@@ -79,4 +80,22 @@ describe('kv', () => {
     const kv = new KeyValueStore()
     expect(() => { kv.delete('hello') }).toThrowError(new Error('HTTP DELETE request failed: 404 Not Found'))
   })
+
+  test('KeyValueStore persistence', async () => {
+    // Create a persistent store
```

Comment:
Could clean up most of these comments? I'd prefer comments be saved for code that is unclear / could be misunderstood. Variable assignments and code that makes sense by reading it shouldn't have comments.